tinkerbell/stack/templates/nginx-configmap.yaml

```diff
@@ -16,7 +16,7 @@ data:
 
     http {
       server {
-        listen 80;
+        listen {{ .Values.boots.http.port }};
```

Comment:
line 26 below will need the explicit port too.

Reply:
Thanks